packages/thirdweb/src/extensions/lens/read/type.ts

```diff
@@ -0,0 +1,51 @@
+export type LensProfileSchema = {
+  $schema: "https://json-schemas.lens.dev/profile/2.0.0.json";
```

Comment:
does this show up in docs or something?

Reply:
hmm not sure about this one. it's there in every profiles's metadata that I pulled up